extensions/api/identity-api/api-configuration/src/main/resources/identity-api-version.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "version": "1.0.0-alpha",
+    "version": "1.1.0-alpha",
```

Comment:
there can't be a `1.1.0-alpha` without there ever having been a `1.0.0`. This should just remain `1.0.0-alpha`, since everything can change at any time in an alpha.

Reply:
If we do not update the version number a workflow fails: https://github.com/eclipse-edc/.github/blob/d3ed5543b6d24916f67f6f020c6ac768dadd647d/.github/workflows/verify-openapi.yml#L60